web-registry/public/_redirects

```diff
@@ -1,5 +1,2 @@
+https://registry.regen.network/*  https://app.regen.network/:splat 301!
 /*    /index.html   200
-
-/projects  /projects/wilmot  200
```

Comment:
removed because this rule would not ever match, because the rule that was before it always matches (`/* /index.html 200`)

Reply:
@mhagel does this look OK to you? see the other similar deletions as well. TL;DR i don't think these redirects were actually doing anything.

Reply:
good catch. That is an old redirect we don't want anymore. This is ok.